docs: introductie voor designers herschreven (#2648)
nl-design-system/documentatie: docs/handboek/designer/introductie.mdx

## Welkom designer 👋

Leuk dat je er bent! Of je nu mee wilt helpen aan NL Design System of simpelweg even wilt rondneuzen, als designer is dit een goed startpunt.

## Wat is NL Design System?

> Het NL Design System zit net even anders in elkaar dan andere designsystemen. Waar veel design systems een specifieke organisatie bedienen, werkt NL Design System met organisaties verspreid over de hele overheid: van gemeenten tot Rijk.

Reviewer: wil je hier 'Het' wel laten staan?

[Meer informatie over NL Design System](/project/over-nl-design-system/)

### Community en kernteam

> Bij NL Design System werken specialisten 'open-source' samen aan betere digitale dienstverlening. Specialisten zijn bijvoorbeeld designers, developers, researchers en toegankelijkheidsspecialisten. Ze werken in hun organisatie aan specifieke webprojecten en delen hun ervaring in de community. Het kernteam faciliteert deze samenwerking.

Reviewer: het valt nu voor mij een beetje weg wie dat kernteam dan is

[Meer informatie over het kernteam](/project/kernteam)

:::info

#### Community sprints

> NL Design System faceliteert ook [Community sprints](https://www.nldesignsystem.nl/community/community-sprints/) waarbij minimaal twee teams uit de community op samenwerken.

Reviewer: ```suggestion
NL Design System faciliteert ook [Community sprints](https://www.nldesignsystem.nl/community/community-sprints/) waar minimaal twee teams uit de community op samenwerken.
```

- [Overheidsbrede portalen community](https://www.nldesignsystem.nl/community/community-sprints/overheidsbrede-portalen-community/). Hier wordt samengewerkt aan portalen voor overheidsorganisaties.
- [Rijkshuisstijl community](https://www.nldesignsystem.nl/community/community-sprints/rijkshuisstijl-community/). Hier wordt samengewerkt aan NL Design System-componenten voor projecten die moeten voldoen aan de Rijkshuisstijl.

:::

### Estafettemodel

Het estafettemodel van NL Design System is een aanpak om samen de beste en meest bruikbare componenten, patronen en richtlijnen te maken.

[Meer informatie over het estafettemodel](/handboek/estafettemodel)

### Huisstijl

> De componenten van het NL Design System hebben van zichzelf geen specifieke huisstijl. Iedere organisatie kan zijn eigen thema op de componenten toepassen. Om dat voor elkaar te krijgen, maken we gebruik van ‘design tokens’.

Reviewer: hier wel ook weer 'het'?

- [Meer informatie over design tokens](/handboek/huisstijl/design-tokens)
- [Meer informatie over thema's](/handboek/huisstijl/themas)

> Vanuit het NL Design System bieden we het '[Start-thema](/handboek/huisstijl/themas/start-thema/)' aan. De stijl van dit Start-thema hebben we vastgelegd in een set aan 'basis-tokens'. Deze basis-tokens leven op het ['Common' niveau](/handboek/huisstijl/design-tokens#common-tokens).

Reviewer: het?

Community componenen, en bijbehorende design tokens, hebben hun eigen prefix. Pas deze niet aan. Zo kan er beter herleidt worden bij welke organisatie de component ontwikkelt is.

:::

- [Componenten overzicht](/componenten)
- [Werken met NL Design System](handboek/designer/werken-met-nl-design-system)
- [Figma hygiëne](/handboek/designer/figma-hygiene)

### Niveau 5. Componenten verbeteren uit de community

Als een component niet bruikbaar is voor jouw organisatie maak dan gebruik van het bestaande ontwerp (of code) om op door te ontwikkelen. Probeer hierbij zoveel mogelijk over te nemen van de bestaande component.

Het is natuurlijk helemaal super als je met de organisatie van de oorspronkelijke component samenwerkt om de verbeteringen door te voeren.

Deel de verbeterde component met de community. Dat kan via Slack, de [GitHub Discussion van de component](https://github.com/orgs/nl-design-system/discussions/categories/component-suggestions) of door een GitHub Issue aan te maken bij de organisatie die de component beheerd.

- [Componenten overzicht](/componenten)
- [Figma hygiëne](/handboek/designer/figma-hygiene)

### Niveau 6. Componenten bijdragen aan de community

> Ontwikkel als organisatie zelf componenten volgens de aanpak van het NL Design System.

Reviewer: het?

## Welkom designer 👋

Leuk dat je er bent! Of je nu mee wilt helpen aan NL Design System of simpelweg even wilt rondneuzen, als designer is dit een goed startpunt.

## Wat is NL Design System?

NL Design System zit net even anders in elkaar. Waar veel design systems een specifieke organisatie bedienen, werkt NL Design System met organisaties verspreid over de hele overheid: van gemeenten tot Rijk.

[Meer informatie over NL Design System](/project/over-nl-design-system/)

### Community en kernteam

> Bij NL Design System werken specialisten uit overheidsorganisaties, van gemeenten tot Rijksoverheid, samen aan betere digitale dienstverlening. Het kernteam faciliteert deze samenwerking. Specialisten zijn bijvoorbeeld ontwerpers, ontwikkelaars en onderzoekers. Ze werken in hun organisatie aan specifieke webprojecten en delen hun ervaring in de community.

Reviewer 2: ```suggestion
Bij NL Design System werken specialisten uit overheidsorganisaties, van gemeenten tot de Rijksoverheid, samen aan betere digitale dienstverlening. Het kernteam faciliteert deze samenwerking. Tot de specialisten behoren bijvoorbeeld ontwerpers, ontwikkelaars en onderzoekers. Zij werken binnen hun eigen organisatie aan specifieke webprojecten en delen hun ervaringen in de community.
```

#### Wat doet de community?

> Specialisten in de community denken bijvoorbeeld samen na over de beste inrichting van een Mijn Zaken-systeem het beste kan worden ingericht, of een toegankelijk meerstappenformulier bouwt. De community ontwikkelt ook de daadwerkelijke componenten en test ze in hun eigen websites en applicaties.

Reviewer 2: ```suggestion
Specialisten in de community denken samen na over vraagstukken, zoals de optimale inrichting van een Mijn Zaken-systeem of het bouwen van een toegankelijk meerstappenformulier. Daarnaast ontwikkelt de community ook de daadwerkelijke componenten en test deze in hun eigen websites en applicaties.
```

#### Wat doet het kernteam?

Het kernteam maakt de samenwerking in de community mogelijk, door bijvoorbeeld de generieke architectuur op te zetten, versiebeheer uit te denken en regie te houden op kwaliteit via het estafettemodel. Ook helpen ze organisaties en hun leveranciers op weg.

[Meer informatie over het kernteam](/project/kernteam)

:::info[Community sprints]

> NL Design System faciliteert ook [Community sprints](https://www.nldesignsystem.nl/community/community-sprints/) waar minimaal twee teams uit de community op samenwerken. Iedereen die met hetzelfde onderwerp bezig is kan zich aanmelden om via deze sprints bij te dragen.

Reviewer 2: ```suggestion
NL Design System faciliteert ook [Community sprints](https://www.nldesignsystem.nl/community/community-sprints/) waar minimaal twee teams uit de community samenwerken aan een gezamenlijk onderwerp. Iedereen die met hetzelfde thema bezig is, kan zich aanmelden om via deze sprints bij te dragen.
```

- [Rijkshuisstijl community](https://www.nldesignsystem.nl/community/community-sprints/rijkshuisstijl-community/). Hier wordt samengewerkt aan NL Design System-componenten voor projecten die moeten voldoen aan de Rijkshuisstijl.

:::

### Estafettemodel

Het estafettemodel van NL Design System is een aanpak om samen de beste en meest bruikbare componenten, patronen en richtlijnen te maken.

[Meer informatie over het estafettemodel](/handboek/estafettemodel)

### Huisstijl

De componenten van NL Design System hebben van zichzelf geen specifieke huisstijl. Iedere organisatie kan zijn eigen thema op de componenten toepassen. Om dat voor elkaar te krijgen, maken we gebruik van ‘design tokens’.

- [Meer informatie over design tokens](/handboek/huisstijl/design-tokens)
- [Meer informatie over thema's](/handboek/huisstijl/themas)

Vanuit NL Design System bieden we het '[Start-thema](/handboek/huisstijl/themas/start-thema/)' aan. De stijl van dit Start-thema hebben we vastgelegd in een set aan 'basis-tokens'. Deze basis-tokens leven op het ['Common' niveau](/handboek/huisstijl/design-tokens#common-tokens).

### Ontwerpprogramma’s

> Of je nu Sketch, Framer of Paint gebruikt, er is geen afhankelijkheid als het gaat om ontwerpprogramma. NL Design System werkt goed met Figma, maar als je een ander ontwerpprogramma gebruikt kijken we graag mee of er een manier is dat design tokens en de componenten opbouw daar ook kan werken.

Author: 'Uitstekend' zou ik ook weer niet zeggen ;) maar ik vind het ook wel weer een mooi woord ter afwisseling waardoor de tekst interessant blijft.

[Meer informatie over Figma structuur en gebruik](/handboek/designer/figma-structuur)

## Meedoen als designer

> Wil je als designer meedoen met NL Design System? Leuk! Je kunt meedoen op op verschillende niveaus.

Reviewer 2: ```suggestion
Wil je als designer meedoen met NL Design System? Leuk! Je kunt op verschillende niveaus instappen en bijdragen.
```

### Niveau 1. Online meedoen

> Doe online mee.

Reviewer 2: ```suggestion
Doe online mee en lever vanuit jouw expertise een bijdrage aan de community.
```

- [Schrijf je in als lid van de community](/community/sluit-je-aan/).
- Deel kennis vanuit je specialisme.
- Deel inzichten die je hebt opgedaan tijdens onderzoek.
- Leg ontwerpkeuzes voor aan specialisten in de community en haal feedback op.
- Stel design gerelateerde vragen waar je wel wat hulp bij kan gebruiken.

Online meedoen kan op verschillende manieren:

> - Sluit aan bij de [Code for NL Slack omgeving](https://praatmee.codefor.nl/). NL Design System heeft hier meerdere Slack kanalen. Specifiek voor designers is dat: [#nl-design-system-designers](https://codefornl.slack.com/archives/C01D78C2E4E). Mag jij geen Slack gebruiken? Dan horen we dat graag [kernteam](/project/kernteam).

Reviewer 2: ```suggestion
- Sluit aan bij de [Code for NL Slack omgeving](https://praatmee.codefor.nl/). NL Design System heeft hier meerdere Slack-kanalen. Specifiek voor designers is dat: [#nl-design-system-designers](https://codefornl.slack.com/archives/C01D78C2E4E). Mag jij geen Slack gebruiken? Laat het weten aan het [kernteam](/project/kernteam).
```

- Sluit aan bij het [Design Open Hour](https://www.nldesignsystem.nl/events/design-open-hour).
- Sluit aan bij de [Heartbeat](https://www.nldesignsystem.nl/events/heartbeat).

> - Laat van je horen via [GitHub Discussions](https://github.com/orgs/nl-design-system/discussions). Hier verzamelen we specifieke informatie over [Componenten](https://github.com/orgs/nl-design-system/discussions/categories/component-suggestions). Maar ook over patronen zoals [Formulieren](https://github.com/orgs/nl-design-system/discussions/categories/formulieren) en [Mijn omgevingen](https://github.com/orgs/nl-design-system/discussions/categories/mijn-omgevingen).

Reviewer 2: Hoofdletter C bij componenten?

Author: Vond ik in dit geval wel gepast.

- Deel je onderzoek op [gebruikersonderzoeken.nl](https://gebruikersonderzoeken.nl/)

:::tip[Tip]

Voor de [Community Sprints](#community-sprints) zijn er specifieke mogelijkheden om online mee te doen.

:::

### Niveau 2. Richtlijnen gebruiken

Onder [Richtlijnen](/richtlijnen) verzamelen we richtlijnen voor het maken van digitale diensten. Zo zijn er bijvoorbeeld richtlijnen voor:

- [Stijl](https://www.nldesignsystem.nl/richtlijnen/stijl)
- [Content](https://www.nldesignsystem.nl/richtlijnen/content)
- [Formulieren](https://www.nldesignsystem.nl/richtlijnen/formulieren)

Maar ook voor elk [WCAG-succescriterium](https://www.nldesignsystem.nl/wcag).

> Ons doel is om praktische uitleg en voorbeelden te geven die helpen bij het toegankelijk inzetten van de NL Design System componenten, patronen en richtlijnen.

Reviewer 2: ```suggestion
Ons doel is om praktische uitleg en concrete voorbeelden te biden die helpen bij het toegankelijk inzetten van NL Design System componenten, patronen en richtlijnen.
```

Author: ```suggestion
Ons doel is om praktische uitleg en concrete voorbeelden te bieden die helpen bij het toegankelijk inzetten van NL Design System componenten, patronen en richtlijnen.
```

> Gebruik de richtlijnen als kennisbank tijdens het ontwerpen en help ons door feedback te geven. Bijvoorbeeld door in Slack te laten weten of de richtlijnen duidelijk zijn en goed werken.

Reviewer 2: ```suggestion
Gebruik de richtlijnen als kennisbank tijdens het ontwerpen en help ons verbeteren door feedback te geven. Bijvoorbeeld door in Slack te laten weten of de richtlijnen duidelijk zijn en goed werken.
```

### Niveau 3. Eigen NL Design System thema maken en onderhouden

Maak een thema voor jouw organisatie door organisatiespecifieke ontwerpbeslissingen vast te leggen in design tokens. Dat doe je door het Figma Stappenplan te volgen.

- [Meer informatie over design tokens](/handboek/huisstijl/design-tokens)
- [Meer informatie over thema's](/handboek/huisstijl/themas)
- [Werken met NL Design System](/handboek/designer/werken-met-nl-design-system)
- [Zelf een thema maken](/handboek/designer/zelf-thema-maken)
- [Figma Stappenplan](/handboek/designer/figma-stappenplan)

### Niveau 4. Componenten hergebruiken uit de community

> Maak voor prototypes, voor zover mogelijk, gebruik van de [Figma bibliotheken van NL Design System](/handboek/designer/figma-structuur). Of herbruik onderdelen van community prototypes.

Reviewer 2: ```suggestion
Maak voor prototypes, waar mogelijk, gebruik van de [Figma bibliotheken van NL Design System](/handboek/designer/figma-structuur). Je kunt ook onderdelen hergebruiken uit community prototypes.
```

> De Figma biblitoheken van NL Design System bevatten veel componenten en worden geregeld aangevuld. De volgende situaties kunnen zich echter voordoen:

Reviewer 2: ```suggestion
De Figma biblitoheken van NL Design System bevatten veel componenten en worden regelmatig aangevuld. Toch kunnen de volgende situaties voorkomen:
```

- De community component die jij nu juist nodig hebt ontbreekt nog. Neem in dat geval contact op met het kernteam zodat we weten dat hier behoefte aan is.

> - De community component is niet bruikbaar. Kies dan je eigen weg. Idealiter deel je de afweging wel met de community. Dat kan via Slack, de [GitHub Discussion van de component](https://github.com/orgs/nl-design-system/discussions/categories/component-suggestions) of door een GitHub Issue aan te maken bij de organisatie die de component beheerd.

Reviewer 2: ```suggestion
- De community component is niet bruikbaar. Kies dan je eigen oplossing, maar deel je afweging bij voorkeur met de community. Dat kan via Slack, de [GitHub Discussion van de component](https://github.com/orgs/nl-design-system/discussions/categories/component-suggestions) of door een GitHub Issue aan te maken bij de organisatie die de component beheert.
```

> - Er zijn verschillende community componenten voor dezelfde toepassing. Gebuik in dat geval de component die het beste aansluit bij de wensen van jouw organisate.

Reviewer 2: ```suggestion
- Er zijn verschillende community componenten voor dezelfde toepassing. Gebuik in dat geval de component die het beste aansluit bij de wensen van jouw organisatie.
```

:::tip[Tip]

> Community componenen, en bijbehorende design tokens, hebben hun eigen prefix. Pas deze niet aan. Zo kan er beter herleidt worden bij welke organisatie de component ontwikkelt is.

Reviewer 2: ```suggestion
Community componenten en bijbehorende design tokens hebben altijd een eigen prefix. Pas deze niet aan, zodat duidelijk blijft bij welke organisatie de component is ontwikkeld.
```

:::

- [Componenten overzicht](/componenten)
- [Werken met NL Design System](/handboek/designer/werken-met-nl-design-system)
- [Figma hygiëne](/handboek/designer/figma-hygiene)

### Niveau 5. Componenten verbeteren uit de community

> Als een component niet bruikbaar is voor jouw organisatie maak dan gebruik van het bestaande ontwerp (of code) om op door te ontwikkelen. Probeer hierbij zoveel mogelijk over te nemen van de bestaande component.

Reviewer 2: ```suggestion
Is een component niet direct bruikbaar voor jouw organisatie? Maak dan gebruik van het bestaande ontwerp (of de code) als basis om verder op door te ontwikkelen. Probeer hierbij zoveel mogelijk elementen van de bestaande component over te nemen.
```

> Het is natuurlijk helemaal super als je met de organisatie van de oorspronkelijke component samenwerkt om de verbeteringen door te voeren.

Reviewer 2: ```suggestion
Het is helemaal super als je samenwerkt met de organisatie die de oorspronkelijke component heeft ontwikkeld, zodat verbeteringen gezamenlijk kunnen worden doorgevoerd.
```

> Deel de verbeterde component met de community. Dat kan via Slack, de [GitHub Discussion van de component](https://github.com/orgs/nl-design-system/discussions/categories/component-suggestions) of door een GitHub Issue aan te maken bij de organisatie die de component beheerd.

Reviewer 2: ```suggestion
Deel de verbeterde component met de community. Dat kan via Slack, de [GitHub Discussion van de component](https://github.com/orgs/nl-design-system/discussions/categories/component-suggestions) of door een GitHub Issue aan te maken bij de organisatie die de component beheert.
```
